.github/workflows/run-k8s-tests-on-aks.yaml

```diff
@@ -47,13 +47,16 @@ jobs:
             vmm: clh
             instance-type: small
             genpolicy-pull-method: oci-distribution
+            auto-generate-policy: yes
```

Comment:
IIUC this will change the job name, so we'll need to un-require the old jobs and re-require the new ones when it gets merged.